ext/mbstring/libmbfl/filters/mbfilter_sjis_2004.c

```diff
@@ -545,6 +545,7 @@ mbfl_filt_conv_wchar_jis2004(int c, mbfl_convert_filter *filter) {
 
 			CK((*filter->output_function)(s1, filter->data));
 			CK((*filter->output_function)(s2, filter->data));
+			s1 = 0;
```

Comment:
Makes sense.

Reply:
Done for SJIS-2004. Same transformation doesn't work for SJIS-mac, since in that case we are falling through from one `switch` case to another, not doing a `goto` to the top of the function.